cc33xx-target-scripts/suite/trixie/debian/control

```diff
@@ -0,0 +1,14 @@
+Source: cc33xx-target-scripts
+Section: utils
+Priority: optional
+Maintainer: Suhaas Joshi <s-joshi@ti.com>
+Build-Depends: debhelper-compat (= 13)
+Standards-Version: 4.5.0
+Homepage: https://www.ti.com/tool/CC33XX-SOFTWARE
+
+Package: cc33xx-target-scripts
+Architecture: arm64
```

Comment:
Are these just scripts? If so they should not be arch dependent.

Reply:
Made them arch-independent.